test/e2e/e2e_test.go

```diff
@@ -642,6 +642,25 @@ func checkMetricsCollection(t *testing.T, promClient prometheusv1.API, namespace
 	}
 }
 
+func checkSignerCAMetrics(t *testing.T, promClient prometheusv1.API) {
+	timeout := 60 * time.Second
+	err := wait.PollImmediate(10*time.Second, timeout, func() (bool, error) {
+		results, warnings, err := promClient.Query(context.Background(), `signer_ca_expiry_time`, time.Now())
```

Comment:
signer_ca -> service_ca

Reply:
renamed instances of signer_ca to service_ca